scripts/serve-static-storybook.js

```diff
@@ -1,13 +1,16 @@
-const StaticServer = require('static-server');
+const handler = require('serve-handler');
```

Comment:
replaced `static-server` package by `serve-handler`. `static-serve` was not serving videos well, and `serve-handler` seems to work fine